arcade/paths.py

```diff
@@ -264,10 +256,4 @@ def astar_calculate_path(start_point: Point,
     graph = _AStarGraph(barrier_list, left, right, bottom, top, diagonal_movement)  # type: ignore
     result = _AStarSearch(mod_start, mod_end, graph)
 
-    if result is None:
-        return None
-
-    # Currently 'result' is in grid locations. We need to convert them to pixel
-    # locations.
-    revised_result = [_expand(p, grid_size) for p in result]
-    return revised_result
+    return None if result is None else [_expand(p, grid_size) for p in result]
```

Comment:
I wonder if this is faster, but it certainly is not an improvement in readability.

Reply:
Not faster but shorter. I like this because for me is concise enough. The list comprehension is short

Reply:
I'm ok with either of these. I'll revert it to the original.